index.html

```diff
@@ -1629,6 +1629,49 @@ <h4>DIDCommMessaging</h4>
 
       </section>
 
+      <section>
+        <h4>CredentialRegistry</h4>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://ssi.eecc.de/api/registry/swagger/#/Credentials/get_api_registry_vcs__id_">Verifiable Credential Registry</a>
```

Comment:
I don't understand the swagger link, can you add a paragraph of text summarizing this?

Reply:
It is a sample of a registry endpoint. I will specify that more precisely :)